apple/LayoutReanimation/REAScreensHelper.h

```diff
@@ -17,5 +18,10 @@
 + (REAUIView *)getScreenWrapper:(REAUIView *)view;
 + (int)getScreenType:(REAUIView *)screen;
 + (bool)isRNSScreenType:(REAUIView *)screen;
++ (bool)isStackChanged:(REAUIView *)view;
++ (REAUIView *)findTopScreenInChildren:(REAUIView *)screen;
++ (REAUIView *)getActiveTabForTabNavigator:(REAUIView *)tabNavigator;
++ (bool)isChild:(REAUIView *)child OfScreen:(REAUIView *)screen;
++ (bool)isOnTop:(REAUIView *)view;
```

Comment:
What does this method do? Can we make its name more descriptive?
```suggestion
+ (bool)isViewOnTopOfScreenStack:(REAUIView *)view;
```

Reply:
I'm not sure if you want to add 'View' here, since it's a strongly typed language.

Reply:
On the other hand it's a recurring pattern in the rest of the code, so maybe it's a good idea.